packages/application-shell/src/components/login-sso/login-sso.js

```diff
@@ -113,75 +105,71 @@ export class LoginSSO extends React.PureComponent {
+                              />
+                            ) : (
+                              formikProps.status.errorMessage
+                            )}
+                          </Text.Body>
+                          {messages[`${errorMessageKey}Secondary`] ? (
+                            <Text.Body>
+                              <FormattedMessage
+                                {...messages[`${errorMessageKey}Secondary`]}
+                              />
+                            </Text.Body>
+                          ) : null}
+                        </Spacings.Stack>
+                      </ContentNotification>
+                    ) : null}
+                    <Text.Subheadline elementType="h3">
+                      <FormattedMessage {...messages.title} />
+                    </Text.Subheadline>
+                    <TextField
                       name="organizationName"
-                      className={loginStyles['input-text']}
-                      value={formikProps.values.organizationName}
-                      onChange={event => {
-                        formikProps.setFieldValue(
-                          'organizationName',
-                          event.target.value
-                        );
-                        formikProps.setFieldTouched('organizationName', true);
-                      }}
-                    />
-                    {formikProps.touched.organizationName &&
-                      formikProps.errors.organizationName && (
-                        <p className={styles.error}>
-                          {formikProps.errors.organizationName}
-                        </p>
+                      title={this.props.intl.formatMessage(
+                        messages.organization
                       )}
-                  </div>
-                  <div>
-                    <PrimaryButton
-                      label={this.props.intl.formatMessage(messages.proceed)}
-                      onClick={formikProps.handleSubmit}
-                      isDisabled={
-                        !formikProps.isValid || formikProps.isSubmitting
-                      }
+                      hint={this.props.intl.formatMessage(
+                        messages.caseSensitive
+                      )}
+                      isAutofocussed={true}
+                      isRequired={true}
+                      value={formikProps.values.organizationName}
+                      touched={formikProps.touched.organizationName}
+                      errors={formikProps.errors.organizationName}
+                      onChange={formikProps.handleChange}
+                      onBlur={formikProps.handleBlur}
+                      isDisabled={formikProps.isSubmitting}
                     />
-                  </div>
-                </Spacings.Stack>
+                    <Spacings.Inline>
+                      {/* FIXME: the <PrimaryButton> current does not behave like a
```

Comment:
Can you open an issue in UI-Kit? This seems like something we should fix and something that could annoy consumers 😇 I imagine the problem is in [AccessibleButton](https://github.com/commercetools/ui-kit/blob/master/src/components/buttons/accessible-button/accessible-button.js), type is set to "button" and not a prop with a defaultValue?

Reply:
See https://github.com/commercetools/ui-kit/issues/340 :)

Reply:
@dferber90 🤦‍♂️ thanks 😄 🎉 🐔